File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_12.go
package main
import ("fmt"
        
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    if (((a > b) && (b > c) && (a > c)) || ((c > b) && (b > a) && (c > a))) {
        y := b
        fmt.Println (y)
    } else if (((b > a) && (a > c) && (b > c)) || ((c > a) && (a > b) && (c > b))) {
        y := a
        fmt.Println (y)
    } else if (((a > c) && (c > b) && (a > b)) || ((b > c) && (c > a) && (b > a))) {
        y := c
        fmt.Println (y)
    } else {
        fmt.Println ("Nekorrektniyi vvod")
    }
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_12.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
if (((a > b) & (b > c)) || ((c > b) & (b > a)))
    y = b
    puts (y)
elsif (((b > a) & (a > c)) || ((c > a) & (a > b)))
    y = a
    puts (y)
elsif (((a > c) & (c > b)) || ((b > c) & (c > a)))
    y = c
    puts (y)
else
    puts ('Necorrectniyi vvod')
end